docs/general-guidance/date-time-specification.md

```diff
@@ -1,11 +1,43 @@
-# Date/Time Specification
+# Date/Time Specification & AIRAC Effective Date
 
-FIXM requires times to be expressed in UTC.
+FIXM requires times to be expressed in UTC. A constraint is therefore placed on the FIXM classes used to represent date/time values that imposes the use of the trailing character `Z` to indicate UTC, in line with the W3C XSD specification.
 
-A constraint is placed on class *Base.Types.Time*, the class used to
-represent all date/time values in FIXM, imposes the use of the trailing
-character `Z` to indicate UTC, in line with the W3C XSD specification.
+Sub-second precision is considered unneeded for most aviation data, but other fields such as message timestamps can still benefit from higher precision date/times.
 
-Example: 20th July 1969 at 20:18 UTC is expressed as `1969-07-20T20:18:00.000Z`
+`FIXM Core 4.3.0` defines two types for representing date/time values, both restricting the standard XML dateTime:
+- `DateTimeUtc` is constrained to only allow whole second precision. It is used for typing all aviation-related times, such as the actual time of arrival.
+- `DateTimeUtcHighPrecision` employs unrestricted sub-second precision. It is used for typing the message timestamps in the FIXM Applications.
+
+Examples: 
+- for aviation-related times, *20th July 1969 at 20:18 UTC* is expressed as `1969-07-20T20:18:00Z`
+
+```xml
+<!--xmlns:fx="http://www.fixm.aero/flight/4.3"-->	
+<fx:actualTimeOfArrival>
+  <fx:time>1969-07-20T20:18:00Z</time>
+```
+
+- for message timestamps, higher precision can be provided, as needed: `1969-07-20T20:18:00.458Z`
```

Comment:
Should we include an example of a DateTimeUtcHighPrecision like the one above for DateTimeUtc, e.g., something like this (from the MessageType class in the Basic Message Application):
<fx:timestamp>
        <fx:time>1969-07-20T20:18:00,458Z</time>

Reply:
Otherwise, I approve of the changes in this pull request